### DISCUSSION TITLE
Login cannot be disabled

### DISCUSSION THREAD

**Discussion by mxcd** [uptrace/uptrace#307] [Announcements]
In the docs (https://uptrace.dev/get/config.html#managing-users) it is stated that "By default, Uptrace allows anyone to use UI to view data".  
However, commenting out the `auth` section in the default `uptrace.yml` config file does not disable the login and I am still required to enter username and password. The suggested default email (`uptrace@localhost`) and password (`uptrace`) are not working either.  
I would expect the login mask to disappear when no auth is configured.

**Comment by vmihailenco** [comment]
That information is outdated. There is currently no way to allow anonymous users to view the data. We don't plan to support such a feature, but feel free to send a PR.

>commenting out the auth section 

This will forbid anyone to login using an email and password. It is needed in case you've configured OpenID.
